Marlin/LED_Strip.cpp

```diff
@@ -0,0 +1,32 @@
+// RGB Led Strip Control
+ 
+#include "Marlin.h"
+#include "temperature.h" 
+#include "Configuration.h"
+#include "ultralcd.h"
+#include "LED_Strip.h"
+
+#if ENABLED(LED_STRIP)
+
+// Set color by RGB values
+void rgb_to_led_strip(int red, int green, int blue) {    
+  analogWrite(REDPIN, red);
+  analogWrite(GREENPIN, green);
+  analogWrite(BLUEPIN, blue);
+}
+
+// Print finished - wait for user click
+void rgb_print_finished() {
```

Comment:
I consider this to be a separate feature, or at least a sub-feature dependent on having one of the color LED options. Let's look at generalizing this so that it can work with `BLINKM` and `RGB_LED`, and perhaps as an option for `TEMP_STAT_LEDS` also.